docs/design/pattern_matching.md

```diff
@@ -157,22 +168,15 @@ fn H(n: i32) {
 }
 ```
 
-As specified in
-[#2022](/proposals/p2022.md#the-behavior-of-unused-name-bindings), the `unused`
-keyword indicates that the binding is intended to not be used.
```

Comment:
I think we should include some text like this from #2022 describing what `unused` does:

> Names that are qualified with the `unused` keyword are visible for name lookup
> but uses are invalid, including when they cause ambiguous name lookup errors. If
> attempted to be used, a compiler error will be shown to the user, instructing
> them to either remove the `unused` qualifier or remove the use.

Reply:
I think this comment is still unresolved.

Reply:
Whoops, missed that. Done.